client/state/ui/selectors.js

```diff
@@ -89,7 +89,7 @@ export function getSectionGroup( state ) {
  * @return {Boolean}       Whether current section is site-specific
  */
 export function isSiteSection( state ) {
-	return includes( [ 'sites', 'editor' ], getSectionGroup( state ) );
+	return includes( [ 'sites', 'editor', 'gutenberg' ], getSectionGroup( state ) );
```

Comment:
Note: this is needed in order to add the site title to the document title (see: https://github.com/Automattic/wp-calypso/blob/master/client/state/document-head/selectors.js#L75)